src/core/simulation_sequence.jl

```diff
@@ -16,6 +16,9 @@ mutable struct SimulationSequence
                                  feed_forward = Dict{Tuple{String, Symbol, Symbol}, AbstractAffectFeedForward}(),
                                  ini_cond_chronology = Dict{String, AbstractChronology}(),
                                  cache = Dict{String, Vector{AbstractCache}}())
+        for (k, interval) in intervals
+            intervals[k] = IS.time_period_conversion(interval)
```

Comment:
It isn't completely clear to me why the conversion is being done here. I saw the code in IS but I can't totally get why it was necessary.

Reply:
So, for / and + datetimes get messed up.... Julia can't automatically handle 1 hour divided by 1 minute..... we talked for a while about the fastest solution and decided that converting every period internally to milliseconds will mean that everything will work together instead of implementing a bunch of conversions or method overloads. I added the print method in IS so that when you do "sim.intervals" it still prints the time nicely in minutes/hours/days instead of milliseconds

Reply:
you can also do `intervals = convert(Dict{String, Dates.Millisecond}, intervals) `

Reply:
I think the benefit of looping over the function is that so if there's ever a time where it's decided "maybe we should do seconds instead of milliseconds" or whatever, than changing the function will change it everywhere @sourabhdalvi